projects/Aerogramme/default.nix

```
@@ -1,9 +1,14 @@
-{ pkgs, ... }:
 {
-  packages = { inherit (pkgs) aerogramme; };
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  name = "Aerogramme";
   nixos = {
     # https://aerogramme.deuxfleurs.fr/documentation/cookbook/config/
     # https://aerogramme.deuxfleurs.fr/documentation/cookbook/service-manager/
+    # derivation at https://github.com/NixOS/nixpkgs/blob/nixos-24.11/pkgs/by-name/ae/aerogramme/package.nix
     modules.services = null;
```

Thinking about it, in order to not lose already existing information we may want an absolutely minimal module here, such as

```suggestion
    modules.services = { lib, pkgs, ... }: {
        options = {
          enable = lib.mkEnableOption "Aerogramme " ;
          package = lib.mkPackageOption pkgs "aerogramme" { };
        }
    };
```